### DISCUSSION TITLE
Bet won't be placed as the amount -4 is less than the minimum required 10

### DISCUSSION THREAD

**Discussion by qwerty123-noname** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#653] [Q&A]
What I must do? I don't understand about bet. Have this in code.
![image](https://user-images.githubusercontent.com/101948786/201754158-242425bb-c3dd-4016-ace9-5f9526421def.png)

**Comment by DIMITREOS** [comment]
Лол у тебя он делает ставки  ? У меня почему то нету
 INFO - [make_predictions]: Place 123 channel points on: Water Rune Enjoyers  (BLUE), Points: 366k, Users: 34 (49.28%), Odds: 2.15 (46.51%)
15/11/22 15:23:26 - ERROR - [post_gql_request]: Error with GQLOperations (MakePrediction): HTTPSConnectionPool(host='gql.twitch.tv', port=443): Max retries exceeded with url: /gql (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025DD3911070>: Failed to establish a new connection: [WinError 10060] ▒▒▒▒▒▒▒ ▒▒▒▒▒▒▒▒▒▒ ▒▒▒▒▒▒▒▒▒▒ ▒▒▒▒ ▒▒▒▒▒▒▒▒▒▒▒, ▒.▒. ▒▒ ▒▒▒▒▒▒▒ ▒▒▒▒▒▒▒▒▒▒ ▒▒ ▒▒▒▒▒▒▒▒▒ ▒▒▒▒▒ ▒▒ ▒▒▒▒▒▒▒ ▒▒▒▒▒▒ ▒▒▒▒▒▒, ▒▒▒ ▒▒▒▒ ▒▒▒▒▒▒▒▒▒ ▒▒▒ ▒▒▒▒▒▒▒▒▒▒▒▒▒ ▒▒▒▒▒▒▒▒▒▒ ▒▒-▒▒ ▒▒▒▒▒▒▒▒▒ ▒▒▒▒▒▒▒ ▒▒▒ ▒▒▒▒▒▒▒▒▒▒▒▒▒ ▒▒▒▒▒▒▒▒▒▒'))

**Reply by qwerty123-noname** [reply]
I have it 
![image](https://user-images.githubusercontent.com/101948786/201983477-4e25c531-33c4-4e1b-b6c9-cd05d2b6dbc7.png)
